frontend/public/locales/zh/workload.json

```diff
@@ -0,0 +1,127 @@
+  "DaemonSets": "DaemonSets",
+  "Start rollout": "开始推出部署",
+  "Resume rollouts": "恢复推出部署",
+  "Pause rollouts": "暂停推出部署",
+  "Latest version": "最新版本",
+  "Message": "消息",
+  "Update strategy": "更新策略",
+  "Timeout": "timeout",
+  "Update period": "更新周期",
+  "Interval": "间隔",
+  "Max unavailable": "最大不可用",
+  "{{maxUnavailable}} of {{count}} pod_0": "{{maxUnavailable}}（共 {{count}}） 个 pod",
+  "Max surge": "最大激增",
+  "{{maxSurge}} greater than {{count}} pod_0": "{{maxSurge}} 大于 {{count}} 个 pod",
+  "Min ready seconds": "最小就绪秒数",
+  "Triggers": "触发器",
+  "DeploymentConfig details": "DeploymentConfig 详情",
+  "Conditions": "条件",
+  "DeploymentConfigs": "DeploymentConfig",
+  "Edit update strategy": "编辑更新策略",
+  "Progress deadline seconds": "进度期限的秒数",
+  "Deployment details": "部署详情",
+  "Deployments": "部署",
+  "resource {{name}}": "资源{{name}}",
+  "(as a percentage of request)": "（作为请求的百分比）",
+  "{{name}} on pods": "{{name}} 在 pod 上",
+  "on": "on",
+  "Metrics": "指标",
+  "Type": "类型",
+  "Current": "当前",
+  "Target": "目标",
+  "(unrecognized type)": "（未识别的类型）",
+  "HorizontalPodAutoscaler details": "HorizontalPodAutoscaler 详情",
+  "Scale target": "扩展目标",
+  "Min replicas": "最小副本",
+  "Max replicas": "最大副本",
+  "Last scale time": "上一次进行缩放的时间",
+  "Current replicas": "当前副本",
+  "Desired replicas": "所需的副本",
+  "Min pods": "最小 pod",
+  "Max pods": "最大 pod",
+  "Edit parallelism": "编辑并行机制",
+  "Job status": "作业状态",
+  "Start time": "开始时间",
+  "Completion time": "完成时间",
+  "Succeeded pods": "调度的 pod",
+  "Active pods": "活跃的 pod",
+  "Failed pods": "失败的 pod",
+  "Completions": "完成",
+  "Ready": "Ready",
+  "Restarts": "重启",
+  "Owner": "所有者",
+  "Node": "节点",
+  "Memory": "内存",
+  "CPU": "CPU",
+  "IP address": "IP 地址",
+  "Image": "镜像",
+  "State": "状态",
+  "Started": "Started",
+  "Finished": "完成",
+  "Exit code": "退出代码",
+  "Memory usage": "内存用量",
+  "CPU usage": "CPU 用量",
+  "Filesystem": "Filesystem",
+  "Network in": "网络入",
+  "Network out": "网络出",
+  "Restart policy": "重启策略",
+  "Pod IP": "Pod IP",
+  "Pod details": "Pod 详情",
+  "Init containers": "Init 容器",
+  "Pods": "Pod",
+  "ReplicaSet details": "ReplicaSet 详情",
+  "Deployment revision": "部署修订",
+  "{{count1}} of {{count2}} pods": "{{count1}}（共{{count2}}）个 pod",
+  "ReplicaSets": "ReplicaSet",
+  "Cancel rollout": "取消推出部署",
+  "Are you sure you want to cancel this rollout?": "您确定要取消此推出部署吗？",
+  "Yes, cancel": "是，取消",
+  "No, don't cancel": "否，不要取消",
+  "ReplicationController details": "ReplicationController 详情",
+  "Phase": "阶段",
+  "{{statusReplicas}} of {{specReplicas}} pods": "{{specReplicas}} 的 {{statusReplicas}} pod",
+  "ReplicationControllers": "ReplicationController",
+  "StatefulSet details": "StatefulSet 详细信息",
+  "StatefulSets": "StatefulSets",
+  "Resource limits": "资源限制",
+  "Ports": "端口",
+  "Remove volume": "删除卷",
+  "Mount path": "挂载路径",
+  "SubPath": "SubPath",
+  "Permissions": "权限",
+  "Utilized by": "被利用"
```

Comment:
Items that could be translated: "Size", "Ready", "Started", "timeout", "second".

Reply:
It looks like this was not addressed.

Reply:
Second and subpath not addressed, rest have been addressed. Tagging @rh-tfu

Reply:
SubPath can leave in English

Reply:
Looks like translations were added for "Size", "Ready", "Started", "timeout", and "second."